subprojects/degrees-and-programs/source/static/.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Static is used for the static source code of the project. Since we also have Cascade files in this project, we wanted to separate them in the file structure this way into `static` and `cascade`. This is typically how we do it in house, and it matches the current structure of our repository.